quarkchain/evm/specials.py

```diff
@@ -237,6 +244,7 @@ def proc_ecpairing(ext, msg):
         b"0000000000000000000000000000000000000006": proc_ecadd,
         b"0000000000000000000000000000000000000007": proc_ecmul,
         b"0000000000000000000000000000000000000008": proc_ecpairing,
+        b"0000000000000000000000000000000000000009": proc_current_mnt_id,
```

Comment:
may want to avoid following the address, since it may be used by future EVM

I would suggest something like `fffffffffffffffffffffffffffffffffffffff0`. what do you think? cc @qizhou